API.md

```diff
@@ -1624,6 +1625,14 @@ Requires the string value to be a valid GUID.
 const schema = Joi.string().guid();
 ```
 
+#### `string.uuid()`
```

Comment:
Could we approach this more like how `Any.only`, `Any.valid`, etc. are handled?

For the header is states: `any.valid(value)` - aliases: `only`, `equal`

You can take a look [here](https://github.com/hapijs/joi/blob/v9.0.4/API.md#anyvalidvalue---aliases-only-equal)